src/metrics/elasticsearch_metric.ts

```diff
@@ -35,7 +35,8 @@ export class ElasticsearchMetric extends AbstractMetric {
     return {
       url: this.datasource.url,
       method: 'POST',
-      schema: { data }
+      schema: { data },
+      headers: {'Content-Type': `application/json;charset=UTF-8`}
```

Comment:
Let's test without this strange charset setting
```suggestion
      headers: {'Content-Type': 'application/json'}
```